wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UIColor+Accent.swift

```diff
@@ -100,6 +100,26 @@ extension UIColor {
         }
     }
 
+    class func lowAccentColorForUsernameMention() -> UIColor {
+        let safeAccentColor = AccentColor(ZMAccentColor: indexedAccentColor()) ?? .blue
+        switch safeAccentColor {
+        case .blue:
+            return SemanticColors.View.backgroundBlueUsernameMention
+        case .red:
+            return SemanticColors.View.backgroundRedUsernameMention
+        case .green:
+            return SemanticColors.View.backgroundGreenUsernameMention
+        case .yellow:
+            return SemanticColors.View.backgroundAmberUsernameMention
```

Comment:
Yellow and amber have the same mention colour?

Reply:
Yep they do. Until we get rid of the yellow accent color from data model since it's deprecated.